### PR TITLE
Skeleton driver for 1995 Accexx 28.8 modem started

### PR DESCRIPTION
**Pull request by JoakimLarsson**
Just a start do get the ROM in there

### CODE REVIEW

File: src/mame/drivers/accexx.cpp [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	MCFG_CPU_ADD("maincpu", I80C31, Y2_CLOCK)
+	MCFG_CPU_PROGRAM_MAP(accexx_map)
+MACHINE_CONFIG_END
+
+ROM_START( accexx )
+	ROM_REGION( 0x10000, "maincpu", 0 )
+	ROM_LOAD( "u16.bin", 0x00000, 0x10000, CRC(36dc423d) SHA1(0f350b7c533eb5270a72587ab3e050e5fe453006) )
+ROM_END
+
+COMP( 1995, accexx,      0,          0,      accexx,      accexx,  driver_device,   0,          "Accexx",        "Accexx",           MACHINE_IS_SKELETON )
```
Comment (cuavas) [inline src/mame/drivers/accexx.cpp]: Could you please add the model number to the name?  I'm pretty sure they made more than one product.

Comment (JoakimLarsson) [inline src/mame/drivers/accexx.cpp]: I was a bit too hasty and got even the name wrong, it is Aceex, I just had the pcb in front of me and it is highly unmarked. Now I found the box and can trace the model down, stay tuned

**Comment by cuavas** [comment]
Isn't this also missing a change to a lua file to make it actually build?

**Comment by Robbbert** [comment]
Need to update scripts/target/mame/mess.lua, and src/mame/mess.flt

**Comment by JoakimLarsson** [comment]
Right, but also for skeletons?  I just wanted it in so it will follow the flow of core changes not be a part of main build yet, but it might have to then. I will add it

**Comment by JoakimLarsson** [comment]
Reason was that I need to write the Rockwell device or at least mock it and it will take a while longer

**Pull request merged**